Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotModule.java

```diff
@@ -0,0 +1,42 @@
+package com.microsoft.tooling.msservices.serviceexplorer.azure.webapp.deploymentslot;
```

Comment:
@sdubov 
a lot of stuff about Deployment slots, maybe we need to add the selector to our web app publish UI?
https://blogs.msdn.microsoft.com/mvpawardprogram/2017/05/16/deploy-app-azure-app-service/